public/app/percona/settings/components/Advanced/Advanced.test.tsx

```diff
@@ -157,7 +173,17 @@ describe('Advanced::', () => {
 
     // expect(spy.calls.mostRecent().args[0].body.stt_check_intervals).toBeUndefined();
     expect(spy).toHaveBeenLastCalledWith(
-      expect.objectContaining({ body: expect.objectContaining({ stt_check_intervals: undefined }) })
+      expect.objectContaining({
+        body: expect.objectContaining({
+          advisor_run_intervals: undefined,
+          data_retention: '6048000s',
```

Comment:
Does it need the additional information outside of the `advisor_run_intervals`?

Reply:
you're right, I've removed the rest of the object parameters